smdebug/tensorflow/tensor_ref.py

```diff
@@ -22,7 +22,9 @@ def get_tf_names(arg):
     elif isinstance(arg, values.DistributedVariable):
         tf_names = [v.name for v in arg._values]
     else:
-        raise NotImplementedError
+        raise NotImplementedError(
```

Comment:
should we just support distributed values I don't see things getting worse than what it is today.
a  proposal is: 
else:
  #log
  if instance of DistributedValue : 
      tf_names = [v.name for v in arg._values] 
 else:
   Not implemented error

Reply:
Done.